### PR TITLE
Add Roman Numerals exercise

### PR DESCRIPTION
**Pull request by Steffan153**

### CODE REVIEW

File: exercises/practice/roman-numerals/roman-numerals_test.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
+-- Setup test table and read in student solution:
+.read ./test_setup.sql
+
+-- Test cases:
+INSERT INTO tests (name, uuid,
+                    number, expected)
```
Comment (IsaacG) [inline exercises/practice/roman-numerals/roman-numerals_test.sql:6]: Is this indentation intentional?

File: exercises/practice/roman-numerals/roman-numerals_test.sql [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        ('3000 is MMM', 'bb550038-d4eb-4be2-a9ce-f21961ac3bc6', 3000, 'MMM'),
+        ('3001 is MMMI', '3bc4b41c-c2e6-49d9-9142-420691504336', 3001, 'MMMI'),
+        ('3888 is MMMDCCCLXXXVIII', '2f89cad7-73f6-4d1b-857b-0ef531f68b7e', 3888, 'MMMDCCCLXXXVIII'),
+        ('3999 is MMMCMXCIX', '4e18e96b-5fbb-43df-a91b-9cb511fe0856', 3999, 'MMMCMXCIX');
+
+-- Comparison of user input and the tests updates the status for each test:
+UPDATE tests
+SET status = 'pass'
+FROM (SELECT number, result FROM "roman-numerals") AS actual
+WHERE (actual.number, actual.result) = (tests.number, tests.expected);
```
Comment (IsaacG) [inline exercises/practice/roman-numerals/roman-numerals_test.sql:40]: Should these lines be indented to match the prior statement?

**Comment by BNAndras** [comment]
Looks okay to me once Isaac’s questions are addressed.

File: exercises/practice/roman-numerals/roman-numerals_test.sql (hunk 2 above)
Comment (Steffan153) [reply exercises/practice/roman-numerals/roman-numerals_test.sql:40]: It's consistent, and if we want to change it, I think it should be a separate PR.

File: exercises/practice/roman-numerals/roman-numerals_test.sql (hunk 1 above)
Comment (Steffan153) [reply exercises/practice/roman-numerals/roman-numerals_test.sql:6]: It's consistent, and if we want to change it, I think it should be a separate PR.

**Review by IsaacG — APPROVED** [review]

**Comment by IsaacG** [comment]
Closing and reopening to bump the checks.

**Comment by IsaacG** [comment]
It looks like you need to resolve some merge conflicts here, @Steffan153

**Pull request merged**